### PR TITLE
Updated main page with buy ticket button that takes user to eventbrit…

### PR DESCRIPTION
**Pull request by miketraverso**
…e page in new tab/window.

### CODE REVIEW

File: app/elements/blocks/tickets-block/tickets-block.html [hunk 1]
```diff
@@ -148,7 +154,8 @@ <h4 class="heading">{$ ticketsBlock.title $}</h4>
                             value: 'buy-a-ticket'
                         }
                     });
-                    this.$.ticketsDialog.open();
+                    window.open('http://www.eventbrite.com/e/gdg-devfest-florida-2016-tickets-27051458641?ref=ebtnebtckt', '_blank');
```
Comment (Splaktar) [inline app/elements/blocks/tickets-block/tickets-block.html]: What's this `ref=ebtnebtckt` doing for us?

**Comment by miketraverso** [comment]
That was the link generated by an eventbrite widget creator. Left it in case that helps eventbrite analytics.

File: app/elements/blocks/tickets-block/tickets-block.html (hunk 1 above)
Comment (Splaktar) [reply app/elements/blocks/tickets-block/tickets-block.html]: Added a new tracking link for the website.

**Comment by Splaktar** [comment]
OK, this is ready to go minus the blog post.

**Comment by Splaktar** [comment]
Deployed this to prod.

**Pull request merged**